lib/templates/contact.html.erb

```diff
@@ -4,8 +4,8 @@
 %>
 <div id="contact_<%= contact.content_id %>" class="<%= ['contact', extra_class].flatten.join(' ') %>">
   <div class="content">
-    <h3><%= contact.title %></h3>
     <div class="vcard contact-inner">
+    <p><%= contact.title %></p>
```

Comment:
I'm intrigued that this has moved position as well as losing the header designation - not saying it's wrong, but what's the reasoning behind it?

Reply:
Ah so, my thinking was that previously it was header, so it was placed _outside_ to indicate that this is different to the `contact-inner` content. However now, this is no longer the case, it's all one-of-the-same semantically so why is it sitting outside?  Also, if there is anything that is applied to `vcard contact-inner` that I can't see this in is update (maybe styling elsewhere?) this update can receive them.  However equally,  happy to update if preferred.

Reply:
It's probably fine either way, but being cautious it might be best to keep the position the same for your last reason! If there is something out there targetting the inner then maybe it's something we wouldn't want this to inherit? But honestly, it probably makes no difference, I'm happy to approve this as is, just wanted to understand if there was reaoning.

Reply:
It's true, both could be true, I think the second part to this ticket might shed a bit more light either way on this one. Hopefully it leans towards solving an issue than causing one, thanks for approving.